Assets/SEE/UI/Window/DesktopWindowSpace.cs

```diff
@@ -64,7 +64,7 @@ private void UpdateActiveTab()
         {
             if (panels.Count > 0 && ActiveWindow != null && ActiveWindow.Window != null)
             {
-                if (!windows.Contains(ActiveWindow))
+                if (!windows.Contains(ActiveWindow) && windows.Count > 0)
```

Comment:
Why is this necessary? Usually, if the `ActiveWindow` is set, it should always be a part of the `windows`.

Reply:
It had fixed a bug that caused WindowSpaces to crash. This now seems to be no longer necessary after your last adjustment.